plugins/woocommerce-blocks/tests/e2e/tests/filter-blocks/active-filters.block_theme.spec.ts

```diff
@@ -1,14 +1,34 @@
 /**
  * External dependencies
  */
-import { test, expect } from '@woocommerce/e2e-playwright-utils';
+import { test as base, expect } from '@woocommerce/e2e-playwright-utils';
+import { Post } from '@wordpress/e2e-test-utils-playwright/build-types/request-utils/posts';
+import path from 'path';
+
+const TEMPLATE_PATH = path.join( __dirname, './active-filters.handlebars' );
+
+const test = base.extend< {
+	defaultBlockPostPage: Post;
```

Comment:
nit: Since it's of type Post, can we call it `defaultBlockPost`?

Reply:
Yep good point, fixed https://github.com/woocommerce/woocommerce/pull/44913/commits/608b666177cad4cac69a3d329eb888d9a33c6efe